packages/node_modules/@webex/webex-core/test/integration/spec/credentials/credentials.js

```diff
@@ -19,6 +19,25 @@ describe('webex-core', () => {
         user = u;
       }));
 
+    describe('#config', () => {
+      let webex;
+
+      it('should accept an authorizationString config property', () => {
```

Comment:
Could you also verify that the `authorizeUrl` is actually being set correctly?

Reply:
updated the `it()` statement to reflect the test that validates `authorizeUrl` is set.